src/components/Api.js

```diff
@@ -0,0 +1,90 @@
+import UserInfo from "./UserInfo";
+
+export default class Api {
+  constructor(options) {
+    // constructor body
+    this._baseUrl = options.baseUrl;
+    this._headers = options.headers;
+  }
+
+  getInitialCards() {
+    return fetch(`${this._baseUrl}/cards`, {
+      headers: this._headers,
+    }).then((res) => {
+      if (res.ok) {
+        return res.json();
+      }
```

Comment:
You need to check every response from the server and invoke an error if the response is not `ok` according to the task


```js

.then(res => {
                if (res.ok) {
                    return res.json();
                }
                return Promise.reject(`Error ${res.status}`);
            })



```